src/windows/export.py

```diff
@@ -1008,6 +1011,16 @@ def accept(self):
             # Reveal done button
             self.close_button.setVisible(True)
 
+            # Restore windows title to show elapsed time
```

Comment:
Perhaps we can create a new function, to prevent duplication of this string formatting?
 `title_message = generate_title_message(seconds, fps)`

Reply:
Yes. The whole way this is done should be simplified. It's just I included this in this unrelated PR and I absolutely didn't want to mix things up.